src/store/actions.js

```diff
@@ -150,17 +110,56 @@ export default {
     dispatch('_setActiveItem', null);
     dispatch('_updateLocal');
   },
-
   _startCreatingItem({ dispatch }) {
     dispatch('_setChangingStatus', true);
   },
 
+  // filters
+
+  _addFilter({ commit, dispatch, getters }, { name, type }) {
+    if (
+      (type === 'tags' || type === 'categories')
```

Comment:
I think that array with these types and .includes would be shorter and easier to read